resources/js/components/NotFound.jsx

```diff
@@ -0,0 +1,13 @@
+// NotFound.jsx
+import React from 'react';
+
```

Comment:
We can use some cool template for this page , https://colorlib.com/wp/free-404-error-page-templates/   this site is only for example fell free to use other